forum/qa-theme/SnowFlat/js/selectLanguage.js

```diff
@@ -0,0 +1,111 @@
+if (typeof CKEDITOR != 'undefined') {
+    CKEDITOR.on( "instanceReady", function() {
+        const codeBlock = document.getElementById( "cke_43" );
+        if ( location.href.includes( "/ask" ) ) {

```

Comment:
Tego IF można chyba wynieść wyżej, bo gdy CKEditor jest gotowy (zdarzenie **"instanceReady"**), to już możesz odczytać **location.href**.